.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,37 +1,11 @@
-Description:
-<!--
-Replace this template with your PR description.
-Please remember to keep in mind the security levels outlined in
-[CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md) and to
-include a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title
-
-* **no**: None
-* **low**: Low chance of potential impact to, or exposure of patient data
-* **moderate**: Moderate chance of potential impact to, or exposure of patient data
-* **severe**: Severe chance of potential impact to, or exposure of patient data
-
-Please also:
-
-* Get thumbs from reviewer(s)
-* Verify all tests go green, including CI tests
--->
-
+**Description**
 
 ---
 **PR checklist**
 
-- [ ] This PR meets the Acceptance Criteria in the JIRA story
-- [ ] The JIRA story has been moved to Dev Review
-- [ ] This PR includes appropriate unit tests
-- [ ] I have added explanatory comments where the logic is not obvious
-- [ ] I have run and tested this change locally, and my testing process is described here
+- [ ] If this PR is associated with a Jira issue, I have included an issue number tag in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md)
+- [ ] I have manually tested this change
+- [ ] This PR includes appropriate automated tests, and I have documented any behavior that cannot be tested with code
 - [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
-- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
```

Comment:
> this PR won't be covered by the CircleCI tests

I don't think this is true, so I've removed this item.

Reply:
Maybe the wording could be improved?

This is intended to convey that the e2e tests that are run pre-merge use the PR's UI but the Test env's API.  So it is true that the PR's API changes won't get tested there.

The main-branch post-merge e2e tests do cover the newest API code.

Reply:
Wow, I thought that we were re-releasing the test API on every PR run. This helps explain why some devs use the test env as a development env.

Why do we even run e2e tests on PR checks if they only check half the code? In some cases they're guaranteed to fail.

I'd like to add an item that describes what the developer should do here. "Run all the tests" would be reasonable if our tests were fast and consistent, but since they aren't that seems inconvenient. "Run all relevant tests" seems like a compromise but requires developers to be loosely aware of all our E2E tests.

What do you do each PR?

Reply:
> Why do we even run e2e tests on PR checks if they only check half the code?
That's a valid way of looking at it, but I think in practice *most* of the tests are relevant *most* of the time.

I like "run all relevant tests".  It's true that this requires a greater awareness of which e2e tests might affect API code, but I think that's a good thing - at least in our current framework.

If I make an API change in a PR, I always also run it locally to test that it's doing what I expect.  I don't necessarily run local e2e tests in the way we're talking about - but maybe I should.

Reply:
> some devs use the test env as a development env.
I would like to explore this some more (not here) because I'm not sure I understand exactly what difference you're describing

Reply:
Re-added with updated wording 👍